spec/unit/support/matchers/be_a_response_like_spec.rb

```diff
@@ -17,90 +17,98 @@
     it 'fails when comparing an integer' do
       expect {
         expect({ 'a' => 1 }).to be_a_response_like({ 'a' => /\d+/ })
-      }.to raise_expectation_not_met_with_keys(bad_keys: ['a'])
+      }.to raise_expectation_not_met_with_summary_parts('! a:', '- /\d+/', '+ 1')
```

Comment:
@ljfranklin I assumed that anyone looking at the unit/spec would be familiar enough with the underlying `be_a_response_like.rb` implementation to know that a `summary` is something it generates, and a closer look shows that a summary part contains one of `-: <dropped key>`, `+ <added key>`, or `! - <old value> + <new value>`. Any suggestions?